src/cheri-pte-ext.adoc

```diff
@@ -43,8 +43,8 @@ clears the tag bit of the capability written to a virtual page with the CW bit
 clear.
 
 NOTE: The implementation of the CW bit does not force a dependency on the tag
-bit's value of the capability written, so implementations must support this
-feature.
+bit's value of the capability written, so implementations are not required
+to this feature.
```

Comment:
I am not sure about this change. The only things in the Zcheri_pte extension are the CW and CD bits in PTEs.  CD is certainly optional and dependent on CW being implemented. However, the idea is that if you support Zcheri_pte is because at the very least the implementation supports CW (but not necessarily CD), so what is the core feature of this extension if CW is optional too?

Reply:
Okay, hopefully fixed by https://github.com/riscv/riscv-cheri/pull/11/commits/264a106f7f72e489baadb62c16f039e37396029c